packages/ifpack2/src/Ifpack2_Details_Chebyshev_def.hpp

```diff
@@ -1561,10 +1561,9 @@ makeTempMultiVector (const MV& B)
   // null, but also if the number of columns match, since some multi-RHS
   // solvers (e.g., Belos) may call apply() with different numbers of columns.
 
-  //W must be initialized to zero when it is used as a multigrid smoother.
```

Comment:
So this statement is simply not correct?

Reply:
I don't think so. Looking at these lines
https://github.com/trilinos/Trilinos/blob/5242864373cc43d2fd8bedfd775c236daddeaf2c/packages/ifpack2/src/Ifpack2_Details_Chebyshev_def.hpp#L1296-L1321
Either `W_` is written as `W := (1/theta)*D_inv*(B-A*X)`, or `W := (1/theta)*D_inv*B`.

But otherwise, we also should have had to zero the vector on every apply, and not just when we first allocate it.